Oops, there's a built in encoder for ROT13

ashleycoxley opened the pull request:
@harperj

ashleycoxley committed:
Oops, there's a built in encoder for ROT13

harperj commented on rot13/main.py:
```diff
@@ -50,7 +17,7 @@ def get(self):
 
 	def post(self):
 		text = self.request.get('text')
```
Interesting, you use tabs!

I use spaces.  It doesn't really matter though.

harperj commented on rot13/main.py:
```diff
@@ -50,7 +17,7 @@ def get(self):
 	def post(self):
 		text = self.request.get('text')
-		encoded_text = rot13_encode(text)
+		encoded_text = text.encode('rot13')
```
Using the builtin is always a good thing.

harperj commented on rot13/main.py:
```diff
@@ -9,39 +9,6 @@
 	autoescape=True)
 
 
-def encode_algo(character):
```
While it's generally the right thing to do to use the standard library, it's nice to see you took the time to implement this while learning.

harperj replied:
I would have named this something like `rot13_endcode_char`

harperj commented on rot13/main.py:
```diff
@@ -9,39 +9,6 @@
-
-	alphabet = 'abcdefghijklmnopqrstuvwxyz'
-	idx = alphabet.index(character)
-	if idx <= 12:
```
This is the same as 

``` python
code_idx = (alphabet.index(character) + 13) % 26
```

Modulus is a nice way to think about rotations.

You could also write it:

``` python
idx = alphabet.index(character)
code_idx =  idx + 13 % alphabet.length
```

harperj replied:
I would call `code_idx` something like `rotated_idx`

harperj commented on rot13/main.py:
```diff
@@ -9,39 +9,6 @@
-		code_idx = idx - 13
-	encoded_char = alphabet[code_idx]
-
-	return(encoded_char, lower_case)
```
You don't need to use parens around return values in Python.

harperj reviewed (approved):
Overall good.  I went ahead and reviewed your rot13 implementation because I thought it might be useful.

ashleycoxley committed:
Corrected style issues

Several changes here:
 - Added style.css to remove inline CSS from form.html
 - Removed unnecessary newlines and print statements from main.py
